Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -67,7 +67,7 @@ inside the `X`-adic completion of `RatFunc K`.
 
 * Since `LaurentSeries` is just an abbreviation of `HahnSeries ℤ _`, the definition of the
 coefficients is given in terms of `HahnSeries.coeff` and this forces sometimes to go back-and-forth
-from `X : _⸨X⸩` to `single 1 1 : HahnSeries ℤ _`.
+from `X : ⸨_⸩` to `single 1 1 : HahnSeries ℤ _`.
```

Comment:
What's going on here? I think the original means
```
from `X : R⸨X⸩` to `single 1 1 : HahnSeries ℤ R`.
```

Reply:
Hm, not sure how I ended up with this, but it's much clearer writing out the `R` rather than using `_`. Fixed